domains/kinddev.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "TeamDarkBot",
+           "email": "",
+           "discord": "1286418351035388006",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.b89WAqaUs2w2qWhy6RcWqiEKuVS23nj9_hBy7BGj4DrYvcLS80qCe9TXIYwr7ZOfowoxhMxo_OZFbBcJY97AQXiLujlwNZihaFtkA3PO2VoENfK8yXH2nrZJHDswgXwyC_C-9nIa_qxznDJwER43RBUsEj1_BerR0T5KkqS6IKp8zS9QQ8eHlMKNfXNkXJoXgs3HYkI29CLC324P-83i9lq70bwhPtbRavRXwfuLvRZ80zeQsKlOE7bJ0VtZADb6nM21zEh4LpgIK7yIvw3NPjgvaNpPh5Uc3nms4aWVv8X0ZFfpqkL8xxaJObdxixT1JfrItfc0QDViY8piiehhoA.ZxNqW900Uvd6tRxg7hRwMg.79Qax7xOn8yBBobN-HDxYQL4Bj7B49Q6aeMH-JewETApREWNJz_y8iC8AI48MhksiYtLJV1FAg6SR-KcVCZ7LkaWzuXQBYVXwUFp_23YDsE.CUejsD_Irwkd1vywdkHiCw"
+        },
+    
+        "record": {
+            "CNAME": "kinddev.is-a.dev"
```

Comment:
A domain can not point to itself

Reply:
> What?

https://www.cloudflare.com/learning/dns/dns-records/dns-cname-record/ Recommend you understand how a CNAME record works in the first place.